msr_version.c

```diff
@@ -11,6 +11,7 @@
 #include <linux/version.h>
 #include <linux/module.h>
 #include <linux/uaccess.h>
+#include "msr_version.h"
```

Comment:
Add new line between standard and local header includes

Reply:
Fixed.